### ISSUE TITLE
Bullets align wrong in Firefox

### ISSUE THREAD

**Issue by opn** [fedwiki/wiki-plugin-markdown#15]
I guess this is a CSS issue? Bullets align in Firefox too far to the left with the text rather than the bullet aligned with the left margin of the paragraph.

<img width="974" alt="screen shot 2015-11-06 at 10 34 16" src="https://cloud.githubusercontent.com/assets/80854/10995086/4d2b5432-8472-11e5-8009-50bc29140ade.png">

While it works fine in Chrome:

<img width="966" alt="screen shot 2015-11-06 at 10 33 41" src="https://cloud.githubusercontent.com/assets/80854/10995087/4d2ccac4-8472-11e5-9ade-65539742673c.png">

**Comment by paul90** [comment]
Simply down to the different browsers having different defaults.

The only real solution, if there is a need for one, is to use a reset css and then build the necessary styles. With the theme support added in the latest version provides a possible route to experiment with this.

**Comment by opn** [comment]
Is the latest theme support version in NPM yet?

**Comment by paul90** [comment]
Yes, it is in wiki v0.8.0

**Comment by opn** [comment]
Ok - I dug into this a bit more, as one of our writers users Firefox and the extensive use of markdown bulleted paragraphs makes it pretty ugly for her on Firefox.

There is a lack of an `<ul>` for the markdown rendered lists. I think possibly by adding this we will make it easier to create a consistent look between browsers?

Below we have bullets rendered with the HTML tag on Firefox:

<img width="732" alt="screen shot 2017-03-27 at 10 53 25" src="https://cloud.githubusercontent.com/assets/80854/24351069/586653c0-12dc-11e7-9bf6-08d074eae15f.png">

And here we have the same text rendered with the markdown plugin - note the absence of `<ul>` tags.

<img width="722" alt="screen shot 2017-03-27 at 10 52 56" src="https://cloud.githubusercontent.com/assets/80854/24351158/930a487e-12dc-11e7-976a-2d8be121ffb0.png">

**Comment by paul90** [comment]
This is simply down to the original markdown plug-in taking a very simplistic approach

A better solution would be to use a markdown library, that produces valid html, as in https://github.com/WardCunningham/small-client-wiki/blob/master/src/lineup.js#L439

**Comment by WardCunningham** [comment]
I will point out that markdown bullet lists make a pretty poor todo list. Better to use individual paragraphs that are easily moved as work gets done, prioritized or deferred to other pages. Unicode offers many marks that work well in paragraphs. ✔︎

I agree with Paul that we will need to clear all css defaults and build up new css specific to a federation style aimed specifically at the unplanned interoperability we expect of a distributed system. I have tried but not mastered css to the level that I would attempt this. If we are to support a proliferation of plugins then this mastery has to be on our roadmap.

I do appreciate the effort Paul put into making the markdown library work in the React version and adhere to the constraints on size and nesting I asserted in the home brew version. I don't recall whether he addressed the checkbox addressing issue required if checkboxes are to be edited by clicking. I copied checklists from GitHub flavored markdown which showcases the interactive nature of both environments. The React version is read-only so this wasn't a requirement.

**Comment by opn** [comment]
Right. But does it not make sense just to patch the missing `<ul>` tag in the Markdown Plugin code? Having a missing HTML tag needs to be fixed either way?

I'm a personally bit worried about the effect on all those bulleted paragraphs I've written - as the HTML versions are indented significantly more. 

@WardCunningham - the markdown bullet lists are rarely used as todo lists. The most common use in my writing, and my experience of writing with other people is simply to create more compact lists of items - and to give them some visual prominence. 

Most commonly it is to highlight a series of links to other pages for instance on a summary page that links to several related sections - table of contents type stuff. For that reason the visual appearance is key - the function is to give a bit of structure and emphasis.

**Comment by WardCunningham** [comment]
I wrote the plugin with the UL in place. I removed it because I didn't want the indent, same as you. The path to uniformity would appear to be well formed html with full control over css with a design that doesn't make a large initial download. I think this is doable, just not done.

Update: I've inspected the commit history out of curiosity. I must be remembering some other personal experience with UL because the code as written would not encourage the recognition of adjacent bullets as members of a single UL. [github](https://github.com/fedwiki/wiki-plugin-markdown/blob/014d093a480f079b1374f6d8969372f17f4e29b9/client/markdown.coffee)
